### PR TITLE
gague and power button component integrated. Integration test written…

### PR DESCRIPTION
**Pull request by Seyi-Toluhi**
… and pass

### CODE REVIEW

**Review by JackCollett — APPROVED** [review]
Talked though with Seyi and Firat, looks great!

File: Frontend/tests/components/buttons.test.jsx
Comment (tomgame984) [inline Frontend/tests/components/buttons.test.jsx:1]: Tests relocated to button-gauge-integration as previously agreed.

File: Frontend/package-lock.json
Comment (tomgame984) [inline Frontend/package-lock.json:1]: Code not reviewed by QE

File: Frontend/package.json
Comment (tomgame984) [inline Frontend/package.json:1]: Code not reviewed by QE

File: Frontend/src/App.jsx
Comment (tomgame984) [inline Frontend/src/App.jsx:1]: Code not reviewed by QE

File: Frontend/src/components/Buttons.jsx
Comment (tomgame984) [inline Frontend/src/components/Buttons.jsx:1]: Code not reviewed by QE

File: Frontend/src/components/Gague.jsx
Comment (tomgame984) [inline Frontend/src/components/Gague.jsx:1]: Code not reviewed by QE

**Review by tomgame984 — APPROVED** [review]

**Pull request merged**